packages/farms/tests/v3.test.ts

```diff
@@ -18,9 +42,15 @@ describe('Config farms V3', () => {
         ChainId.ARBITRUM_ONE,
         ChainId.LINEA,
         ChainId.BASE,
+        ChainId.OPBNB,
```

Comment:
Let's use `supportedChainId` from farms

Reply:
do we need to check testnet?

Reply:
we can filter out testnets

Reply:
maybe add a `isTestnet` utility function to chains

Reply:
done